examples/inverse/plot_lcmv_beamformer_volume.py

```diff
@@ -61,10 +61,16 @@
 data_cov = mne.compute_covariance(epochs, tmin=0.04, tmax=0.15,
                                   method='shrunk')
 
-# Run free orientation (vector) beamformer. Source orientation can be
-# restricted by setting pick_ori to 'max-power' (or 'normal' but only when
-# using a surface-based source space)
-stc = lcmv(evoked, forward, noise_cov, data_cov, reg=0.05, pick_ori=None)
+# Run free orientation (vector) beamformer with weight normalization (neural
+# activity index, NAI). Providing a noise covariance matrix enables whitening
+# of the data and forward solution. Source orientation is optimized by
+# setting pick_ori to 'max-power'.
+# weight_norm can also be set to 'unit-noise-gain'. Source orientation can also
+# be 'normal' (but only when using a surface-based source space) or None,
+# which computes a vector beamfomer. Note, however, that not all combinations
+# of orientation selection and weight normalization are implemented yet.
+stc = lcmv(evoked, forward, noise_cov, data_cov, reg=0.05,
+           pick_ori='max-power', weight_norm='nai')
```

Comment:
The range of output from NAI is different, so the cutoff value for plotting changed. Plus, this example now runs the orientation selection, before it computed the vector beamformer. So yes, slightly different output is expected.